dev-portal/package.json

```diff
@@ -1,25 +1,33 @@
 {
-  "name": "dev-portal",
```

Comment:
I feel like we should keep it named dev-portal and bump the version (next line)?

Reply:
Oh, whoops. Remnant of upgrading. Will do.